core/src/main/java/org/frankframework/senders/FrankSender.java

```diff
@@ -0,0 +1,360 @@
+				}
+			}
+			return plr.getResult();
+		};
+	}
+
+	@Nonnull
+	Adapter findAdapter(String target) throws SenderException {
+		AdapterManager actualAdapterManager;
+		String adapterName;
+		int configNameSeparator = target.indexOf('/');
+		if (configNameSeparator > 0) {
+			adapterName = target.substring(configNameSeparator + 1);
+			String configurationName = target.substring(0, configNameSeparator);
+			Configuration configuration = ibisManager.getConfiguration(configurationName);
+			if (configuration == null) {
+				throw new SenderException(getLogPrefix()+"Configuration [" + configurationName + "] not found");
+			}
+			actualAdapterManager = configuration.getAdapterManager();
+		} else if (configNameSeparator == 0) {
+			adapterName = target.substring(1);
+			actualAdapterManager = adapterManager;
+		} else {
+			adapterName = target;
+			actualAdapterManager = adapterManager;
+		}
+		Adapter adapter = actualAdapterManager.getAdapter(adapterName);
+		if (adapter == null) {
+			throw new SenderException(getLogPrefix() + "Cannot find adapter specified by [" + target + "]");
+		}
+		return adapter;
+	}
+
+	Scope determineActualScope(@Nullable ParameterValueList pvl) {
+		ParameterValue scopeParam = pvl != null ? pvl.findParameterValue(SCOPE_PARAM_NAME) : null;
+		if (scopeParam != null) {
+			return Scope.valueOf(scopeParam.asStringValue(getScope().name()));
+		}
+		return getScope();
+	}
+
+	String determineActualTarget(@Nullable ParameterValueList pvl) {
+		ParameterValue targetParam = pvl != null ? pvl.findParameterValue(TARGET_PARAM_NAME) : null;
+		if (targetParam != null) {
+			return targetParam.asStringValue(getTarget());
+		}
+		return getTarget();
+	}
+
+	/**
+	 * Synchronous or Asynchronous execution of the call to other adapter or system.
+	 * <br/>
+	 * Set to <code>false</code> to make the call asynchronously. This means that the current adapter
+	 * continues with the next pipeline and the result of the sub-adapter that was called, or other system that was called,
+	 * is ignored. Instead, the input message will be returned as the result message.
+	 *
+	 * @ff.default true
+	 */
+	public void setSynchronous(boolean b) {
+		synchronous = b;
+	}
+
+	/**
+	 * {@link Scope} decides if the FrankSender calls another adapter, or another Java program running in the same JVM.
+	 * <br/>
+	 * It is possible to set this via a parameter. If the parameter is defined but the value at runtime
+	 * is empty, then the value set via this attribute will be used as default.
+	 *
+	 * @param scope Either {@code ADAPTER}, {@code  JVM} or {@code DLL}.
+	 *
+	 * @ff.default ADAPTER
+	 */
+	public void setScope(Scope scope) {
+		this.scope = scope;
+	}
+
+	/**
+	 * Target: service-name of service in other application that should be called, or name of adapter to be called.
+	 * If the adapter is in another configuration, prefix the adapter name with the name of that configuration and a "/".
```

Comment:
Kunnen we meer nadruk leggen op de "/" ?

Reply:
I hope it is better now?